modules/gapi/misc/python/shadow_gapi.hpp

```diff
@@ -27,13 +27,28 @@ namespace cv
        GAPI_WRAP void setInput(const std::string& name, const cv::GFrame& value);
    };
 
+   class GAPI_EXPORTS_W_SIMPLE GInferListInputs
+   {
+   public:
+       GAPI_WRAP GInferListInputs();
+       GAPI_WRAP void setInput(const std::string& name, const cv::GArray<cv::GMat>& value);
+       GAPI_WRAP void setInput(const std::string& name, const cv::GArray<cv::Rect>& value);
```

Comment:
where's the implementation?

Reply:
I only see .hpp files in this MR

Reply:
There is no implementation, this file isn't compiled at all.

Reply:
It's just a trigger for bindings generator

Reply:
But why do we need it then? How it works? Can you please comment it in the code so then strangers like me wouldn't ask those questions again and again?